apps/web-giddh/src/app/vouchers/list/list.component.ts

```
@@ -2473,4 +2578,526 @@ export class VoucherListComponent implements OnInit, OnDestroy {
+     *
+     * @return {*}  {void}
+     * @memberof VoucherListComponent
+     */
+    public onSubmit(): void {
+        this.formToSave = cloneDeep(this.settingResponse);
+        this.formToSave.invoiceSettings = cloneDeep(this.settingForm.get('invoiceSettings').value);
+        this.formToSave.estimateSettings = cloneDeep(this.settingForm.get('estimateSettings').value);
+        this.formToSave.proformaSettings = cloneDeep(this.settingForm.get('proformaSettings').value);
+        if (this.urlVoucherType === 'purchase') {
+            this.formToSave.purchaseBillSettings = cloneDeep(this.settingForm.get('purchaseBillSettings').value);
+            delete this.formToSave.purchaseBillSettings.invoiceSettings;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)